tools/staticbuild/windows_cd/buildtool/mxnet-distro/doc/CU100MKL_ADDITIONAL.md

```diff
@@ -0,0 +1,27 @@
+Prerequisites
```

Comment:
let's try to deduplicate these files by using the ones in the repo: https://github.com/apache/incubator-mxnet/blob/master/tools/pip/doc/CU102_ADDITIONAL.md